luigi_pipeline/configs/seqr-mito-loading-local.cfg

```diff
@@ -4,19 +4,22 @@
 sample_type = WGS
 genome_version = 38
 dont_validate=True
-source_paths = ["test_ten_cmg_mito_combined.mt"]
+source_paths = ["gs://seqr-datasets/cmg_mito/test_ten_cmg_mito_combined.mt"]
 dest_path = variants.mt
-reference_ht_path = combined_reference_data_chrM.ht
-clinvar_ht_path = clinvar.GRCh38.2022-02-27.ht
+reference_ht_path = gs://seqr-reference-data/GRCh38/mitochondrial/all_mito_reference_data/combined_reference_data_chrM.ht
+clinvar_ht_path = gs://seqr-reference-data/GRCh38/clinvar/clinvar.GRCh38.ht
 high_constraint_interval_path = gs://seqr-reference-data/GRCh38/mitochondrial/Helix high constraint intervals Feb-15-2022.tsv
 
 [SeqrMitoGenotypesMTTask]
 sample_type=WGS
 genome_version=38
-source_paths = ["test_ten_cmg_mito_combined.mt"]
-remap_path = gs://seqr-datasets/v02/GRCh38/RDG_WGS_Broad_Internal/base/projects/R0486_cmg_gcnv/R0486_cmg_gcnv_remap.tsv
-subset_path = gs://seqr-datasets/v02/GRCh38/RDG_WGS_Broad_Internal/base/projects/R0486_cmg_gcnv/R0486_cmg_gcnv_ids.txt
+source_paths = ["gs://seqr-datasets/cmg_mito/test_ten_cmg_mito_combined.mt"]
 dest_path = genotypes.mt
```

Comment:
Looks like everything got moved to buckets except the output, is that the desired behavior?

Reply:
It is an intermediate output. So it is acceptable to be local.